Modules/posixmodule.c

```diff
@@ -10422,19 +10422,24 @@ os_lockf_impl(PyObject *module, int fd, int command, Py_off_t length)
 os.lseek -> Py_off_t
 
     fd: int
+        An open file descriptor, as returned by os.open().
     position: Py_off_t
-    how: int
+        Position, interpreted relative to 'whence'.
+    whence as how: int
+        The relative position to seek from. Valid values are:
+        - SEEK_SET: seek from the start of the file.
+        - SEEK_CUR: seek from the current file position.
+        - SEEK_END: seek from the end of the file.
     /
 
-Set the position of a file descriptor.  Return the new position.
+Set the position of a file descriptor, and return the new position.
 
-Return the new cursor position in number of bytes
-relative to the beginning of the file.
+The return value is the number of bytes relative to the beginning of the file.
```

Comment:
I'm not sure this is needed, but perhaps explaining that 'position' is measured in bytes is useful?

Reply:
I think it is ok, since this is a low-level API.